src/parsers/manifest/dash/node_parsers/Period.ts

```diff
@@ -43,6 +48,8 @@ export interface IPeriodAttributes {
   start? : number;
   duration? : number;
   bitstreamSwitching? : boolean;
+  linkURL? : string;
+  resolveAtLoad? : boolean;
```

Comment:
node_parsers should very minimally parse the manifest here.

I would guess we should just have somethings like `xlinkHref: string` and `xlinkActuate: string`

Reply:
(Even if not much is done here). This is just to be able to easily evolve when we refacto or add features.

Node_parsers just takes the XML and translate it to aa JSON with translated durations etc.
Then the parser takes this JSONified representation and parse it into a IParsedManifest.

This reduce the cognitive burden of having to memorize what we do with what. We just name it the same way and "basta".